src/resolver.js

```
@@ -44,12 +63,34 @@ export class Resolver {
     }
 }
 
-
+/* Create a resolver.
+ */
 export function createResolver(options) {
     return new Resolver(options);
 }
 
 
+/* Create a strict resolver.
+ *
+ * A strict resolver must define an authorize and a mask function.
+ */
+export function createStrictResolver(options) {
+    if (!options.aggregate) {
+        throw new Error('Strict resolver must define an `aggregate` option');
+    }
+    if (!options.authorize) {
+        throw new Error('Strict resolver must define an `authorize` option');
+    }
+    if (!options.mask) {
```

if we don't specify a mask we end up using the default mask, since we never skip masking I would think that this wouldn't be needed. wdyt?

---

nvm, @pelegdvir wrote the same thing